contracts/vaults/BaseVaults/base/RibbonVault.sol

```diff
@@ -168,6 +174,7 @@ contract RibbonVault is
         GAMMA_CONTROLLER = _gammaController;
         MARGIN_POOL = _marginPool;
         GNOSIS_EASY_AUCTION = _gnosisEasyAuction;
+        UNISWAP_ROUTER = _uniswapRouter;
```

Comment:
require(_uniswapRouter != address(0), "_uniswapRouter");

Reply:
Added this require.